tests/conftest.py

```diff
@@ -150,6 +156,11 @@ def _bdb(_setup_database, _configure_bigchaindb):
     flush_db(conn, dbname)
 
 
+@pytest.fixture
+def _genesis(_bdb, genesis_block):
```

Comment:
> Another thought was to separate out the genesis marker from bdb, so that it was obvious you needed both (and would error if you only have genesis?), but this might be more work than worth.

Well, yeah, that could be one approach. The idea with the `genesis` marker in this PR is to have a quick way to require a full setup; including a genesis block, on top of which tests can be written.

The `inputs*` fixtures do require an explicit call to the `bdb` marker though.

I think that as time goes by and we write new tests and use these markers, we'll see the pros and cons, and can adapt and change things accordingly.